common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Settings.scala

```diff
@@ -9,6 +8,7 @@ object IndexSettings {
     "type" -> "custom",
     "tokenizer" -> "standard",
     "filter" -> Json.arr(
+      "english_possessive_stemmer",
```

Comment:
Maybe - I'm not sure it matters. Though I do agree its generally a good idea to convert to lowercase early. Do you have a specific reason?